source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -945,6 +960,38 @@ void InjaTransformer::transform(Http::RequestOrResponseHeaderMap &header_map,
   } else if (merged_extractors_to_body_) {
     std::string output = json_body.dump();
     maybe_body.emplace(output);
+  } else if (merge_templates_.size() > 0) {
+
+    for (const auto &merge_template : merge_templates_) {
+      const std::string &name = std::get<0>(merge_template);
+      
+      // prepare variables for non-advanced_templates_ scenario
+      absl::string_view name_to_split;
+      json* current = nullptr;
+      // if (!advanced_templates_) {
```

Comment:
If we don't end up supporting json pointer templates, I think we should keep this check in and throw in the constructor at config time

Reply:
I need to understand better what would be entailed, I honestly don't really know the different 🙈

Reply:
It's largely just handling of `/` and `~`

spec: https://datatracker.ietf.org/doc/html/rfc6901

our fork to re-introduce support in inja: https://github.com/solo-io/inja/pull/6

Reply:
I'm now throwing an exception at config time